.docker-env/.env

```diff
@@ -3,3 +3,4 @@ DESCRIPTION="A simple DJ server running in Docker"
 REPOSITORY=examples/configs
 REDIS_CACHE=redis://query_broker:6379/0
 CELERY_BROKER=redis://query_broker:6379/1
+QUERY_SERVICE=http://host.docker.internal:8001
```

Comment:
This won't work on linux unless we add the below to the `dj` service in the docker compose file.
```yaml
services:
  dj:
    ...
    extra_hosts:
      - "host.docker.internal:host-gateway"
```
Something about how `host.docker.internal` is specific to the docker desktop apps for mac and windows.

Reply:
Ah yes, good catch! Let me add that